### ISSUE TITLE
Code and config file to train on nuScenes and SemanticKITTI

### ISSUE THREAD

**Issue by zzxxtt** [Cavendish518/SFPNet#2]
Hello,

Thank you for your outstanding project and for making it open source! I noticed that while you have provided the script and configuration files for S.MID, there are no equivalent files for nuScenes and SemanticKITTI. To help the community reproduce the results more effectively, could you please share the code and configuration files for training on nuScenes and SemanticKITTI?

Thank you for your support!

**Comment by Cavendish518** [comment]
Thank you for your attention to our work. Unfortunately, due to the requirements of related projects, we will not fully release the code until 2025.04. However, you can easily use the core module code we provide in the model folder and the training code of our baseline [SphereFormer](https://github.com/dvlab-research/SphereFormer/tree/master) to reproduce the results of the paper. As stated in the paper, SFPM can replace the window attention module in the existing network.

**Comment by Cavendish518** [comment]
We provide some tips to help you reproduce the code. 
1. Code part: You can directly replace and adjust the content under the model file in [SphereFormer](https://github.com/dvlab-research/SphereFormer/tree/master) .
2. Model part: The hyperparameters of SFPM are consistent on all datasets, which is also the original intention of our research. You can refer to the config file that has been provided.
3. Training part: For all datasets, we used Focal loss, trained for 70 epochs, and the basic learning rate was 0.0008 (nuscenes) and 0.001 (Kitti). The learning rate scaling ratio of the SFPM part is 0.4. Because we were more concerned about the ability of the model itself to extract features, we did not spend a lot of effort on parameter adjustment. We encourage you to try more combinations. 

I hope these can help you in your research. Thanks!

**Comment by zzxxtt** [comment]
Thank you for the provided detail, I will try it.
